p2p/proto/transaction.proto

```diff
@@ -66,19 +72,18 @@ message Transaction
         repeated         Felt252 calldata = 6;
     }
 
+    // see https://external.integration.starknet.io/feeder_gateway/get_transaction?transactionHash=0x29fd7881f14380842414cdfdd8d6c0b1f2174f8916edcfeb1ede1eb26ac3ef0
     message DeployAccountV3 {
-        Felt252          max_fee          = 1;
-        AccountSignature signature        = 2;
-        Hash             class_hash       = 3;
-        Felt252          nonce            = 4;
-        Felt252          address_salt     = 5;
-        repeated         Felt252 calldata = 6;
-        ResourceLimits   l1_gas           = 7;
-        ResourceLimits   l2_gas           = 8;
-        Felt252          tip              = 9;
-        Address          paymaster        = 10;
-        string           nonce_domain     = 11;
-        string           fee_domain       = 12;
+        AccountSignature signature        = 1;
+        Hash             class_hash       = 2;
+        Felt252          nonce            = 3;
+        Felt252          address_salt     = 4;
+        repeated         Felt252 calldata = 5;
+        ResourceBounds   resource_bounds  = 6;
+        Felt252          tip              = 7;
+        Address          paymaster_data   = 8;
+        string           nonce_domain     = 9;  // rename to nonce_data_availability_mode ?
+        string           fee_domain       = 10; // rename to fee_data_availability_mode ?
     }
 
     message InvokeV0 {
```

Comment:
Some old invokes are incompatible, maybe we should add optional `entry_point_type`

```
  message InvokeV0 {
    starknet.common.Felt252 max_fee = 1;
    AccountSignature signature = 2;
    starknet.common.Address address = 3;
    starknet.common.Felt252 entry_point_selector = 4;
    repeated starknet.common.Felt252 calldata = 5;
    optional EntryPointType entry_point_type = 6;
  }

  enum EntryPointType {
    EXTERNAL = 0;
    L1_HANDLER = 1;
  }
```